plugins/kosync.koplugin/main.lua

```diff
@@ -361,8 +360,8 @@ function KOSync:login()
                 },
             },
         },
-        width = math.floor(Screen:getWidth() * 0.8),
-        height = math.floor(Screen:getHeight() * 0.4),
+        width = math.floor(Device.screen:getWidth() * 0.8),
```

Comment:
Could you stick with the general coding style? ^_^

local Device = require("device")
local Screen = Device.screen

(less sure about that DeviceModel thing, that seems like a one-off)